Significant Slash to New Tortuga's Tribute Amount

McloughlinGuy opened the pull request:
## Summary
I slashed New Tortuga's tribute amount down to a sixth of it's original amount due to how obnoxiously high it was. I'm open to changing the amount if it seems too low, but I'd like to keep it down below greenrock's amount of 1800 credits. Thanks to @Arachi-Lover for pointing this out on discord.

## Testing Done
I tested multiple different planets to gauge their tribute with my fleet of various assorted warships. I found that with New Tortuga zero got killed, with Greenrock one got killed, and with Rust, surprisingly, two got killed. Due to how strong rust is I bumped it all the way down to 1000 credits. I'd say that it needs to be below Greenrock's tribute amount for sure, but I'm definitely open to changing the value if it feels too low.

McloughlinGuy commented:
Note: I tested with a fleet of 12 different heavy warships, from various stages of the game, from a protector all the way to a hurricane. It was a pretty strong fleet but it was still able to be damaged by the defense fleets of Rust and Greenrock. I'll provide my save file if anyone wants it.

Arachi-Lover commented:
This is currently paying higher than Foundry (more than 2x in fact, 6000 vs 2900), the Syndicate HQ, "where more than half of the commercial freighters in human space are manufactured" so there is no excuse from the part of there being mention of some manufacturing there.
It's also incongruous that this'd be the one Pirate system willing to pay such a high value. Greenrock most likely sees more money than this, being the hub of all piracy in the South, and a major slave market to boot. Are they just not as willing? Why would New Tortuga have this unusually high concentration of pirate lords willing to pay a lot more tribute?

As is, the tribute is ludicrously high, and most inappropriate.

Zitchas commented:
I'm not saying that it shouldn't change, I'm just saying that this needs a bit more thought than "it was an easier fight than Greenock, so it must pay less." 

Also, the slave trade is not a wildly profitable industry, generally.

McloughlinGuy commented:
> I'm not saying that it shouldn't change, I'm just saying that this needs a bit more thought than "it was an easier fight than Greenock, so it must pay less."

I did say I was open to change. The amount I put was an initial amount I gauged based on the difficulty of the fights, it could be changed. Keep in mind 1000 is still on the high end for pirate planets.

Zitchas commented on data/map planets.txt:
```diff
@@ -2725,7 +2725,7 @@ planet "New Tortuga"
 	bribe 0.03
 	security 0
-	tribute 6000
+	tribute 1000
```
```suggestion
	tribute 1500
```


Based on the arguments presented, I'd say somewhere in the 1500-2000 range would be good.

Saugia replied:
I suppose I could see 1500, but I'm leaning towards 1400. I feel that's as high as I'd go here.

Zitchas replied:
Given how extreme 6000 is, I'd really appreciate someone digging up why it was set that high to begin with. I doubt we accidentally set its tribute to double the value of the Syndicate capital. It's quite possible that the Syndicate capital is on the low side; but also just as possible that Tortuga is the secret home of the off-shore bank accounts that will pay generously to keep people from sticking their noses where they don't belong.

I'd like to hear the thoughts of the people working on the pirate campaign/rework here. Possibly the Syndicate people too.

This just seems too obvious to be just a mistake. Although admittedly, it is possible.

Saugia replied:
This is a planet that was added to the game through Exile content, so it was either done by Raven or Lake. I'll request them.

warp-core replied:
It's been at 6000 since the first commit that added it to that branch. Never edited.

ravenshining replied:
It was set by Nomadic Volcano

Zitchas reviewed (changes requested):
I'd like to check the git blame to see why 6000 was picked as an initial starting point, but that is... difficult... to do now. Maybe someone with better git skills can do so.

I'd also point out that New Tortuga probably is home to a fairly vast amount of crime happening in the Syndicate areas. Basically the "Off shore banking" location like those that make so much more money that the rest of their country here on Earth.

Hecter94 reviewed (approved):
This change as-is looks good to merge. I see no reason to block this based on it not being conclusively proven that it wasn't intended.
If it turns out it was intended, that can be done in its own PR with justification for why it should be as high as it is.

ravenshining reviewed (changes requested):
New Tortuga should be a disproportionately hard target for story reasons.  I agree with Zitchas's compromise, and/or increasing the fleets.

Arachi-Lover commented:
Story reason or not, increasing the fleets to justify a 6000 pay would mean upping the # of fleets/difficulty of individual fleets by something like a factor of 4, which'd be pretty ridiculous as that'd dwarf every other pirate world, so set to the 1500-2000 is the cleaner option to me.

McloughlinGuy commented on data/map planets.txt:
```diff
@@ -2725,7 +2725,7 @@ planet "New Tortuga"
 	bribe 0.03
 	security 0
-	tribute 6000
+	tribute 1000
```
```suggestion
	tribute 1400
```

McloughlinGuy commented:
It won't let me request 2 people at once and removed @Galaucus review request when I tried whoops
I changed it to 1400 since that seems to be a good amount.

Zitchas reviewed (approved):
Still a little on the low side, I think it should be >=1500, not below it.

For the comparison to Greenrock. Which is likely to have a larger cash flow: A hub of the slave trade and crime across a bunch of low-population fringe planets; or a major freeport that serves the criminal and grey/black market needs of a collection of heavily populated industrial worlds & humanity's largest corporation? Yes, it's not the only one, but based on its outfitters it's a fairly major one, and it explicitly calls out the fact that they make a living manufacturing things the Syndicate deems illegal... But that the Syndicate isn't willing to actually come out this far to put a stop to it.

Given that it's really not that far out. Certainly not farther than the Syndicate could *easily* manage logistics of the occasional raid. To me, that suggests the Syndicate may well be turning a blind eye to it, which implies that it is providing something of value to the Syndicate (or at least its leadership). That could range from being a quite place where upper management could actually be kings over small domains, to someplace where they can take part in their illegal vice of choice without being disturbed by the police; or merely a place to park their mansion where absolutely *no one* can tell them what to do or take it away.

Approving because 1400 is close enough and this is a rather minor detail. But I would be much happier with it a bit higher.
